### PR TITLE
Tutorials layout

### LINKED ISSUE #9867: [Docs] Create Tutorials layout
### Desired State

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Notes for Reviewers**

This PR fixes #9867 
-  init tutorials collection

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 7d74df6e4e7e42de23109fe9ee3d4e127fa4aed8 at: https://659be7f45fbe6952a078c177--meshery-docs-preview.netlify.app

**Review by leecalcote — CHANGES_REQUESTED** [review]
The actual layout is missing.

Should "tutorials" be located under "guides"?

**Comment by leecalcote** [comment]
> The actual layout is missing.
> 
> Should "tutorials" be located under "guides"?

I see the WIP label now...

**Comment by leecalcote** [comment]
ETA?

File: docs/_config_dev.yml [hunk 1]
```diff
@@ -95,6 +95,7 @@ collections:
     output: true
     permalink: reference/mesheryctl/:collection/:path
     name: mesheryctlcommands
+
```
Comment (vishalvivekm) [inline docs/_config_dev.yml:98]:
```suggestion
```

**Comment by l5io** [comment]
🚀 Preview for commit d105b19e3381765b031e3294a8899548b2d6f4d2 at: https://65a53ee42043d4151577fdcd--meshery-docs-preview.netlify.app

**Comment by vishalvivekm** [comment]
@leecalcote 
For now, I've used two of the random tutorials post just so that I can show the output :
- on landing page of Tutorials:  [guides/tutorials](https://65a549a6f790771f3a646268--meshery-docs-preview.netlify.app/guides/tutorials), tutorials will be listed model wise ( using model frontmatter variable) : 

image

- Each individual tutorial, will also have suggested-reading at the bottom: 
image

If this looks good, I'll remove the two post and then we can merge this for now. @leecalcote

File: docs/_layouts/tutorials.html
Comment (leecalcote) [inline docs/_layouts/tutorials.html:1]: I wonder if you might remove this until its necessary?

or if you might add a permanent "Related Tutorials" section that would then justify a dedicated layout.

Comment (vishalvivekm) [reply docs/_layouts/tutorials.html:1]: Let's keep it for now, I'll come back to this.

**Pull request merged**